rhacs-operator/tasks/main.yml

```diff
@@ -0,0 +1,106 @@
+---
+- name: Setting Up Results Var
+  ansible.builtin.set_fact:
+    test_result: "pass"
+# 1. Testing RHACS operator
+- name: Testing Red Hat Advanced Cluster Security
+  block:
+    - name: Create stackrox namespace
+      kubernetes.core.k8s:
+        name: stackrox
+        api_version: v1
+        kind: Namespace
+        state: present
+    - name: Create stackrox central services
+      kubernetes.core.k8s:
+        name: stackrox-central-services
+        namespace: stackrox
+        api_version: platform.stackrox.io/v1alpha1
+        kind: Central
+        definition:
+          metadata:
+            labels:
+              opdev: opcap
+          spec:
+            central:
+              exposure:
+                route:
+                  enabled: true
+        wait: true
+        wait_condition:
+          status: "True"
+          type: "Deployed"
+        wait_timeout: 90
+        state: present
+    - name: Get Central Secret Password
+      ansible.builtin.shell: |
+        /usr/local/bin/oc -n stackrox get secret central-htpasswd -o go-template={% raw %}'{{index .data "password" | base64decode}}'{% endraw %}
```

Comment:
This shouldn't be necessary. You can use the k8s_info module to get the same data.

Reply:
You're right it's done.